test/common/ssl/test_data/password_protected_cert.cfg

```diff
@@ -0,0 +1,36 @@
+[req]
```

Comment:
Could you reuse the existing `san_uri_cert.cfg` instead of making another copy of the same file?

Reply:
Addressed in 0b2d916.